gulpfile.js

```diff
@@ -44,8 +44,8 @@ gulp.task('build', (done) => {
 });
 
 gulp.task('compile', () => {
-  const srcs = gulp.src(['src/**/*.ts', 'custom_typings/**/*.ts']);
-  const tsResult = srcs.pipe(sourcemaps.init())
+  const tsResult = gulp.src(['src/**/*.ts', 'custom_typings/**/*.ts'])
```

Comment:
Arg, we should really be getting sources from the the tsproject

(ok really really we should be dropping gulp)

Reply:
You somehow picked up a commit I removed before making the PR. Weird... This commit's already merged.